cmd/tenant-service/main.go

```diff
@@ -98,11 +106,21 @@ func main() {
 	}
 	updateLoggingSettings(log)
 
-	cfgViper.WatchConfig()
-	cfgViper.OnConfigChange(func(e fsnotify.Event) {
+	csmViper.WatchConfig()
+	csmViper.OnConfigChange(func(e fsnotify.Event) {
 		updateLoggingSettings(log)
 	})
 
+	ticker := time.NewTicker(time.Second)
```

Comment:
Same as my previous comment -- should this code block be removed?

Reply:
Removed.